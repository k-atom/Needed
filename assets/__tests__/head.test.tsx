import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router'

import Header from '../ts/components/header'


const SetSidebarOpen = (open: boolean) => {
  console.log(open)
};


test('Header deafault', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Header SetSidebarOpen={SetSidebarOpen} />
    </MemoryRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
