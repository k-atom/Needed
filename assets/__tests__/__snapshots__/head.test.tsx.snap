// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header deafault 1`] = `
<nav
  className="navbar navbar-light head__navbar"
>
  <div
    className="navbar-brand icon-block"
  >
    <i
      className="material-icons icon ripple"
      onClick={[Function]}
    >
      menu
    </i>
  </div>
  <a
    className="navbar-brand"
    href="/"
    onClick={[Function]}
    title="Needed"
  >
    <img
      alt="Needed Logo"
      className="head__logo"
      src="/images/K_Logo_160x160.png"
    />
    Needed
  </a>
</nav>
`;
